src/services/listCourtCases.ts

```diff
@@ -33,6 +33,10 @@ const listCourtCases = async (
     query.andWhere("courtCase.errorCount > 0")
   }
 
+  if (urgentFilter) {
```

Comment:
Mind just chucking an integration test in for this functionality?

Reply:
Added in [10af358](https://github.com/ministryofjustice/bichard7-next-ui/pull/95/commits/10af3582ecc0119d508095fef861b5f89786fb60)